bedrock/mozorg/templates/mozorg/about/leadership/includes/executive.html

```diff
@@ -289,7 +289,7 @@ <h4 class="fn" itemprop="name">Steve Teixeira</h4>
         <p class="title" itemprop="jobTitle">Chief Product Officer</p>
 
         <ul class="elsewhere">
-          <li><a class="url twitter" itemprop="url" rel="me external" href="https://twitter.com/stevetex">@stevetex</a>
+          <li><a class="url twitter" itemprop="url" rel="me external" href="https://twitter.com/stevetex" aria-label="X (formerly Twitter)">@stevetex</a>
```

Comment:
I'm not sure `aria-label` is really needed here either, since the link already has an accessible name? If we don't think the accessible name gives enough context, then we could just update the link text to benefit everybody?

Reply:
Yeah the link itself doesn't necessarily make it obvious that a person is hovering on a social link when using a screenreader, it just mentions @ and the username. We could update it similarly to how we have it for our socials in the footer where the text is. 

I can do something like this, and I've tested it using a screenreader, and it'll have the `@include visually-hidden` property set on it. What do you think?

```
 <li><a class="url twitter" itemprop="url" rel="me external" href="https://twitter.com/stevetex"><span class="elsewhere-website">X (formerly Twitter): </span>@stevetex</a></li>
```

Reply:
This seems like a good improvement to me 👍